data/reusables/dependabot/default-dependencies-allow-ignore.md

```diff
@@ -1 +1 @@
-By default all dependencies that are explicitly defined in a manifest or lock file are kept up to date. You can use `allow` and `ignore` to customize which dependencies to maintain with version updates. {% data variables.product.prodname_dependabot %} checks for all allowed dependencies and then filters out any ignored dependencies or versions. So a dependency that is matched by both an `allow` and an `ignore` will be ignored.
+By default all dependencies that are explicitly defined in a manifest are kept up to date. You can use `allow` and `ignore` to customize which dependencies to maintain with version updates. {% data variables.product.prodname_dependabot %} checks for all allowed dependencies and then filters out any ignored dependencies or versions. So a dependency that is matched by both an `allow` and an `ignore` will be ignored.
```

Comment:
```suggestion
By default all dependencies that are explicitly defined in a manifest are kept up to date, but note that if you have security updates enabled, it will also create updates for vulnerable dependencies only included in lock files. You can use `allow` and `ignore` to customize which dependencies to maintain with version updates. {% data variables.product.prodname_dependabot %} checks for all allowed dependencies and then filters out any ignored dependencies or versions. So a dependency that is matched by both an `allow` and an `ignore` will be ignored.
```

Reply:
I'm not sure how to best clarify this, but I made some initial suggestion!

Reply:
Thanks for clarifying the behavior here. I think this might be better as two sentences, how about:

```suggestion
By default all dependencies that are explicitly defined in a manifest are kept up to date by {% data variables.product.prodname_dependabot %} version updates. In addition, {% data variables.product.prodname_dependabot %} security updates also update vulnerable dependencies that are defined in lock files. You can use `allow` and `ignore` to customize which dependencies to maintain. {% data variables.product.prodname_dependabot %} checks for all allowed dependencies and then filters out any ignored dependencies or versions. So a dependency that is matched by both an `allow` and an `ignore` will be ignored.
```

Reply:
I think that's better, yeah!